packages/flutter_tools/test/general.shard/runner/flutter_command_test.dart

```diff
@@ -160,24 +160,19 @@ void main() {
       );
       await flutterCommand.run();
 
-      verify(usage.sendCommand(
-        'dummy',
-        parameters: anyNamed('parameters'),
-      ));
-      verify(usage.sendEvent(
-        'tool-command-result',
-        'dummy',
-        label: 'success',
-        parameters: anyNamed('parameters'),
-      ));
-      expect(verify(usage.sendEvent(
+      expect(usage.events, equals(<TestUsageEvent>[
```

Comment:
FYI if provided with an object that does not implement `Matcher`, the `expect` function will wrap the object in an equals matcher.

Reply:
Oh yeah, I knew that. I think I was playing with `unorderedEquals` and forgot to remove it totally.